### ISSUE TITLE
Run on MacOS 11.1 fails with SuitableContextNotFound error

### ISSUE THREAD

**Issue by lukesutton** [QuantumBadger/Speedy2D#9]
Just testing out a simple starter app using the sample code included in the README. Unwrapping the result of `Window::new_centered` panics with the error SuitableContextNotFound.

I did also experiment with building a window without vsync, but the results were the same.

**Comment by lukesutton** [comment]
Here is the backtrace:

```
BacktraceError { error: SuitableContextNotFound, backtrace:    0: backtrace::backtrace::libunwind::trace
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56/src/backtrace/libunwind.rs:90:5
      backtrace::backtrace::trace_unsynchronized
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56/src/backtrace/mod.rs:66:5
   1: backtrace::backtrace::trace
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56/src/backtrace/mod.rs:53:14
   2: backtrace::capture::Backtrace::create
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56/src/capture.rs:176:9
   3: backtrace::capture::Backtrace::new
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56/src/capture.rs:140:22
   4: speedy2d::error::BacktraceError<E>::new
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/error.rs:84:28
   5: speedy2d::window::WindowImpl<UserEventType>::new::{{closure}}
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/window.rs:387:17
   6: core::option::Option<T>::ok_or_else
             at ~/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/option.rs:561:25
   7: speedy2d::window::WindowImpl<UserEventType>::new
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/window.rs:385:30
   8: speedy2d::Window<UserEventType>::new_with_user_events
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/lib.rs:930:27
   9: speedy2d::Window::new_with_options
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/lib.rs:916:9
  10: ui_test::main
             at src/main.rs:25:11
  11: core::ops::function::FnOnce::call_once
             at ~/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
  12: std::sys_common::backtrace::__rust_begin_short_backtrace
             at ~/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/std/src/sys_common/backtrace.rs:125:18
  13: std::rt::lang_start::{{closure}}
             at ~/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/std/src/rt.rs:66:18
  14: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/ops/function.rs:259:13
      std::panicking::try::do_call
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:379:40
      std::panicking::try
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:343:19
      std::panic::catch_unwind
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panic.rs:396:14
      std::rt::lang_start_internal
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/rt.rs:51:25
  15: std::rt::lang_start
             at ~/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/std/src/rt.rs:65:5
  16: _main
, cause: None }
```

**Comment by QuantumBadger** [comment]
Sorry to hear that, I'll look into this.

Out of interest, what kind of Mac are you using, and which type of GPU?

**Comment by QuantumBadger** [comment]
Also if you have the full log please (including the bit where it's trying different multisampling levels) that would be very helpful.

**Comment by lukesutton** [comment]
I'm running: iMac (Retina 4K, 21.5-inch, Late 2015)
The GPU is: Intel Iris Pro Graphics 6200 1536 MB

I've rerun the demo code from the examples directory and got this log:

```
inished dev [unoptimized + debuginfo] target(s) in 1m 10s
     Running `target/debug/examples/hello_world`
2021-03-02 07:47:55,504 DEBUG [speedy2d::window] Monitor #0 (primary): Monitor #41010
2021-03-02 07:47:55,505 INFO  [speedy2d::window] Trying vsync=true, multisampling=8...
2021-03-02 07:47:55,619 TRACE [winit::platform_impl::platform::view] Triggered `viewDidMoveToWindow`
2021-03-02 07:47:55,620 TRACE [winit::platform_impl::platform::view] Completed `viewDidMoveToWindow`
2021-03-02 07:47:55,620 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,620 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,621 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,621 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,621 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,621 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,625 TRACE [winit::platform_impl::platform::window] Locked shared state in `set_fullscreen`
2021-03-02 07:47:55,625 TRACE [winit::platform_impl::platform::window] Unlocked shared state in `set_fullscreen`
2021-03-02 07:47:55,626 TRACE [winit::platform_impl::platform::window] Dropping `UnownedWindow` (0x7fa501008710)
2021-03-02 07:47:55,626 INFO  [speedy2d::window] Failed with error: OpenGlVersionNotSupported
2021-03-02 07:47:55,626 INFO  [speedy2d::window] Trying vsync=true, multisampling=4...
2021-03-02 07:47:55,632 TRACE [winit::platform_impl::platform::view] Triggered `viewDidMoveToWindow`
2021-03-02 07:47:55,632 TRACE [winit::platform_impl::platform::view] Completed `viewDidMoveToWindow`
2021-03-02 07:47:55,632 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,632 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,632 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,632 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,632 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,632 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,633 TRACE [winit::platform_impl::platform::window] Locked shared state in `set_fullscreen`
2021-03-02 07:47:55,633 TRACE [winit::platform_impl::platform::window] Unlocked shared state in `set_fullscreen`
2021-03-02 07:47:55,739 TRACE [winit::platform_impl::platform::window] Dropping `UnownedWindow` (0x7fa501007760)
2021-03-02 07:47:55,739 INFO  [speedy2d::window] Failed with error: OpenGlVersionNotSupported
2021-03-02 07:47:55,739 INFO  [speedy2d::window] Trying vsync=true, multisampling=2...
2021-03-02 07:47:55,744 TRACE [winit::platform_impl::platform::view] Triggered `viewDidMoveToWindow`
2021-03-02 07:47:55,744 TRACE [winit::platform_impl::platform::view] Completed `viewDidMoveToWindow`
2021-03-02 07:47:55,745 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,745 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,745 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,745 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,745 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,745 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,745 TRACE [winit::platform_impl::platform::window] Locked shared state in `set_fullscreen`
2021-03-02 07:47:55,745 TRACE [winit::platform_impl::platform::window] Unlocked shared state in `set_fullscreen`
2021-03-02 07:47:55,749 TRACE [winit::platform_impl::platform::window] Dropping `UnownedWindow` (0x7fa4ffd432a0)
2021-03-02 07:47:55,749 INFO  [speedy2d::window] Failed with error: OpenGlVersionNotSupported
2021-03-02 07:47:55,749 INFO  [speedy2d::window] Trying vsync=true, multisampling=1...
2021-03-02 07:47:55,754 TRACE [winit::platform_impl::platform::view] Triggered `viewDidMoveToWindow`
2021-03-02 07:47:55,754 TRACE [winit::platform_impl::platform::view] Completed `viewDidMoveToWindow`
2021-03-02 07:47:55,754 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,754 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,754 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,754 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,754 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,754 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,754 TRACE [winit::platform_impl::platform::window] Locked shared state in `set_fullscreen`
2021-03-02 07:47:55,754 TRACE [winit::platform_impl::platform::window] Unlocked shared state in `set_fullscreen`
2021-03-02 07:47:55,758 TRACE [winit::platform_impl::platform::window] Dropping `UnownedWindow` (0x7fa4ffc845a0)
2021-03-02 07:47:55,758 INFO  [speedy2d::window] Failed with error: OpenGlVersionNotSupported
2021-03-02 07:47:55,758 INFO  [speedy2d::window] Trying vsync=true, multisampling=0...
2021-03-02 07:47:55,763 TRACE [winit::platform_impl::platform::view] Triggered `viewDidMoveToWindow`
2021-03-02 07:47:55,763 TRACE [winit::platform_impl::platform::view] Completed `viewDidMoveToWindow`
2021-03-02 07:47:55,763 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,763 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,763 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,763 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,763 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,763 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,763 TRACE [winit::platform_impl::platform::window] Locked shared state in `set_fullscreen`
2021-03-02 07:47:55,763 TRACE [winit::platform_impl::platform::window] Unlocked shared state in `set_fullscreen`
2021-03-02 07:47:55,767 TRACE [winit::platform_impl::platform::window] Dropping `UnownedWindow` (0x7fa4ffcb4550)
2021-03-02 07:47:55,767 INFO  [speedy2d::window] Failed with error: OpenGlVersionNotSupported
2021-03-02 07:47:55,767 INFO  [speedy2d::window] Trying vsync=false, multisampling=8...
2021-03-02 07:47:55,772 TRACE [winit::platform_impl::platform::view] Triggered `viewDidMoveToWindow`
2021-03-02 07:47:55,772 TRACE [winit::platform_impl::platform::view] Completed `viewDidMoveToWindow`
2021-03-02 07:47:55,772 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,772 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,772 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,772 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,772 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,772 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,772 TRACE [winit::platform_impl::platform::window] Locked shared state in `set_fullscreen`
2021-03-02 07:47:55,772 TRACE [winit::platform_impl::platform::window] Unlocked shared state in `set_fullscreen`
2021-03-02 07:47:55,776 TRACE [winit::platform_impl::platform::window] Dropping `UnownedWindow` (0x7fa4ffd326f0)
2021-03-02 07:47:55,777 INFO  [speedy2d::window] Failed with error: OpenGlVersionNotSupported
2021-03-02 07:47:55,777 INFO  [speedy2d::window] Trying vsync=false, multisampling=4...
2021-03-02 07:47:55,781 TRACE [winit::platform_impl::platform::view] Triggered `viewDidMoveToWindow`
2021-03-02 07:47:55,781 TRACE [winit::platform_impl::platform::view] Completed `viewDidMoveToWindow`
2021-03-02 07:47:55,781 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,781 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,781 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,782 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,782 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,782 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,782 TRACE [winit::platform_impl::platform::window] Locked shared state in `set_fullscreen`
2021-03-02 07:47:55,782 TRACE [winit::platform_impl::platform::window] Unlocked shared state in `set_fullscreen`
2021-03-02 07:47:55,787 TRACE [winit::platform_impl::platform::window] Dropping `UnownedWindow` (0x7fa4ffe5b3d0)
2021-03-02 07:47:55,787 INFO  [speedy2d::window] Failed with error: OpenGlVersionNotSupported
2021-03-02 07:47:55,787 INFO  [speedy2d::window] Trying vsync=false, multisampling=2...
2021-03-02 07:47:55,792 TRACE [winit::platform_impl::platform::view] Triggered `viewDidMoveToWindow`
2021-03-02 07:47:55,792 TRACE [winit::platform_impl::platform::view] Completed `viewDidMoveToWindow`
2021-03-02 07:47:55,792 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,792 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,792 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,792 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,792 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,792 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,792 TRACE [winit::platform_impl::platform::window] Locked shared state in `set_fullscreen`
2021-03-02 07:47:55,792 TRACE [winit::platform_impl::platform::window] Unlocked shared state in `set_fullscreen`
2021-03-02 07:47:55,797 TRACE [winit::platform_impl::platform::window] Dropping `UnownedWindow` (0x7fa4ffdbb2a0)
2021-03-02 07:47:55,797 INFO  [speedy2d::window] Failed with error: OpenGlVersionNotSupported
2021-03-02 07:47:55,797 INFO  [speedy2d::window] Trying vsync=false, multisampling=1...
2021-03-02 07:47:55,801 TRACE [winit::platform_impl::platform::view] Triggered `viewDidMoveToWindow`
2021-03-02 07:47:55,801 TRACE [winit::platform_impl::platform::view] Completed `viewDidMoveToWindow`
2021-03-02 07:47:55,802 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,802 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,802 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,802 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,802 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,802 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,803 TRACE [winit::platform_impl::platform::window] Locked shared state in `set_fullscreen`
2021-03-02 07:47:55,803 TRACE [winit::platform_impl::platform::window] Unlocked shared state in `set_fullscreen`
2021-03-02 07:47:55,808 TRACE [winit::platform_impl::platform::window] Dropping `UnownedWindow` (0x7fa4ffcf3610)
2021-03-02 07:47:55,808 INFO  [speedy2d::window] Failed with error: OpenGlVersionNotSupported
2021-03-02 07:47:55,808 INFO  [speedy2d::window] Trying vsync=false, multisampling=0...
2021-03-02 07:47:55,813 TRACE [winit::platform_impl::platform::view] Triggered `viewDidMoveToWindow`
2021-03-02 07:47:55,813 TRACE [winit::platform_impl::platform::view] Completed `viewDidMoveToWindow`
2021-03-02 07:47:55,813 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,813 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,813 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,813 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,813 TRACE [winit::platform_impl::platform::view] Triggered `validAttributesForMarkedText`
2021-03-02 07:47:55,813 TRACE [winit::platform_impl::platform::view] Completed `validAttributesForMarkedText`
2021-03-02 07:47:55,813 TRACE [winit::platform_impl::platform::window] Locked shared state in `set_fullscreen`
2021-03-02 07:47:55,813 TRACE [winit::platform_impl::platform::window] Unlocked shared state in `set_fullscreen`
2021-03-02 07:47:55,818 TRACE [winit::platform_impl::platform::window] Dropping `UnownedWindow` (0x7fa4ffcf8820)
2021-03-02 07:47:55,818 INFO  [speedy2d::window] Failed with error: OpenGlVersionNotSupported
2021-03-02 07:47:55,818 ERROR [speedy2d::window] Failed to create any context.
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: BacktraceError { error: SuitableContextNotFound, backtrace:    0: backtrace::backtrace::libunwind::trace
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56/src/backtrace/libunwind.rs:90:5
      backtrace::backtrace::trace_unsynchronized
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56/src/backtrace/mod.rs:66:5
   1: backtrace::backtrace::trace
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56/src/backtrace/mod.rs:53:14
   2: backtrace::capture::Backtrace::create
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56/src/capture.rs:176:9
   3: backtrace::capture::Backtrace::new
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.56/src/capture.rs:140:22
   4: speedy2d::error::BacktraceError<E>::new
             at src/error.rs:84:28
   5: speedy2d::window::WindowImpl<UserEventType>::new::{{closure}}
             at src/window.rs:387:17
   6: core::option::Option<T>::ok_or_else
             at ~/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/option.rs:561:25
   7: speedy2d::window::WindowImpl<UserEventType>::new
             at src/window.rs:385:30
   8: speedy2d::Window<UserEventType>::new_with_user_events
             at src/lib.rs:930:27
   9: speedy2d::Window::new_with_options
             at src/lib.rs:916:9
  10: speedy2d::Window::new_centered
             at src/lib.rs:887:9
  11: hello_world::main
             at examples/hello_world.rs:30:18
  12: core::ops::function::FnOnce::call_once
             at ~/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
  13: std::sys_common::backtrace::__rust_begin_short_backtrace
             at ~/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/std/src/sys_common/backtrace.rs:125:18
  14: std::rt::lang_start::{{closure}}
             at ~/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/std/src/rt.rs:66:18
  15: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/ops/function.rs:259:13
      std::panicking::try::do_call
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:379:40
      std::panicking::try
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:343:19
      std::panic::catch_unwind
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panic.rs:396:14
      std::rt::lang_start_internal
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/rt.rs:51:25
  16: std::rt::lang_start
             at ~/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/std/src/rt.rs:65:5
  17: _main
, cause: None }', examples/hello_world.rs:30:76
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by QuantumBadger** [comment]
Thanks!

It looks like the issue was that macOS was unhappy with us explicitly requesting a Core context for GL 2.0: https://github.com/QuantumBadger/Speedy2D/commit/a0a3513ccc3d722c88b8f0d5488f7d3fdb4462de

I've published v1.0.4, please let me know if that fixes the issue for you. The fix works for me on a (cloud rented :smile:) Mac Mini, so I'll mark this as resolved.

**Comment by lukesutton** [comment]
@QuantumBadger It works! Thank you 👍
